### ISSUE TITLE
Index selection checkboxes not toggling in web ui

### ISSUE THREAD

**Issue by aaronsdevera** [simon987/sist2#247]
**Device Information (please complete the following information):**
 - OS: `Mac OS10.15.7`
 - Browser: `Chrome 97.0.4692.99`
 - SIST2 Version: `v2.11.6`
 
**Describe the bug**
Checkboxes for indices in the web ui fail to toggle

**Steps To Reproduce**
1. Go to sist web ui
2. Click on checkbox for an index

**Expected behavior**
Selection of checkbox to filter on which indices to select in current search.

**Actual Behavior**
Selection of checkbox do not toggle, thus not filtering on the indices.

**Screenshots**
![ezgif-3-b8ab1f049b](https://user-images.githubusercontent.com/5042762/150859991-6a7e46aa-5e4e-454b-a9af-b246ba372205.gif)

**Additional context**
Here is the traceback in the browser debug console.
```
index.js:94 
TypeError: Cannot read properties of undefined (reading 'shiftKey')
    at a.toggleIndex (index.js:1:55349)
    at change (index.js:1:53562)
    at ne (chunk-vendors.js:7:11664)
    at a.n (chunk-vendors.js:7:13484)
    at ne (chunk-vendors.js:7:11664)
    at a.t.$emit (chunk-vendors.js:7:26491)
    at a.<anonymous> (chunk-vendors.js:81:103675)
    at Array.<anonymous> (chunk-vendors.js:7:12761)
    at ue (chunk-vendors.js:7:12156)
```

**Comment by dpieski** [comment]
jw, does it work if "Update the Media Types tree in real time" is deselected?

**Comment by simon987** [comment]
Thanks!, I think that the last shift-click update (#206) broke the checkbox buttons. For now you can just click on the list item itself

**Comment by aaronsdevera** [comment]
> jw, does it work if "Update the Media Types tree in real time" is deselected?

looks like the issue persists

**Comment by simon987** [comment]
Fixed in c6820b6cc621b8b8003f6e208b4daa7ba5ab8c16
